docs/api/ref/Events.md

```diff
@@ -76,16 +150,63 @@ interface FDC3ChannelChangedEvent extends FDC3Event {
   };
 }
 ```
+</TabItem>
+<TabItem value="dotnet" label=".NET">
+
+```csharp
+public interface IFdc3ChannelChangedEventDetails
+{
+    string? CurrentChannelId { get; }
+}
+public class Fdc3ChannelChangedEventDetails : IFdc3ChannelChangedEventDetails
+{
+    public string? CurrentChannelId { get; }
+
+    public Fdc3ChannelChangedEventDetails(string? channelId)
+    {
+        this.CurrentChannelId = channelId;
+    }
+}
+
+public class Fdc3ChannelChangedEvent : Fdc3Event
+{
+    public Fdc3ChannelChangedEvent(string? channelId)
+        : base(Fdc3EventType.UserChannelChanged, new Fdc3ChannelChangedEventDetails(channelId))
+    {
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
 
 Type representing the format of `userChannelChanged`  events.
 
 The identity of the channel joined is provided as `details.currentChannelId`, which will be `null` if the app is no longer joined to any channel.
 
 ## `PrivateChannelEventTypes`
 
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript/JavaScript">
+
 ```ts
 type PrivateChannelEventTypes = "addContextListener" | "unsubscribe" | "disconnect";
 ```
+</TabItem>
+<TabItem value="dotnet" label=".NET">
+
+```csharp
+public static class Fdc3PrivateChannelEventType
+{
+    public const string AddContextListener = "AddContextListener";
+    public const string Unsubscribe = "Unsubscribe";
+    public const string Disconnect = "Disconnect";
```

Comment:
The PR to fdc3-dotnet was just updated based on feedback.  For developers using the literals rather than the constants themselves, it makes sense to match across implementations for convenience.

```suggestion
    public const string AddContextListener = "addContextListener";
    public const string Unsubscribe = "unsubscribe";
    public const string Disconnect = "disconnect";
```